test/helpers/oauth/facebook/graph-api.js

```diff
@@ -128,8 +128,9 @@ class GraphAPI {
     if (!Array.isArray(permissions) || permissions.length === 0) {
       const users = data.filter((x) => !x.access_token);
       const user = users[users.length * Math.random() | 0];
+
       if (!user) {
-        return this.getTestUserWithPermissions(permissions, nextPage);
+        return this._getTestUserWithPermissions(permissions, paging.next);
```

Comment:
not sure that it is correct (`_getTestUserWithPermissions` instead of `getTestUserWithPermissions`)